packages/hooks/src/useWithdrawLocked.ts

```diff
@@ -0,0 +1,12 @@
+import { useStore } from '@deriv/stores';
+import useNeedPOI from './useNeedPOI';
+
+const useWithdrawLocked = () => {
+    const { client } = useStore();
+    const is_need_poi = useNeedPOI();
+
+    if (!client.account_status?.status) return false;
```

Comment:
```suggestion
```
@hamid-deriv I think we can skip this check 🤔

Reply:
Sorted.